pastes/views.py

```diff
@@ -1,3 +1,45 @@
+import json
 from django.shortcuts import render
+from django.http import HttpResponse
+from django.shortcuts import redirect
 
-# Create your views here.
+from .models import Pastes
+from . import tokens
+
+def list(request):
```

Comment:
y'all need generic views in your lives :)

https://docs.djangoproject.com/en/2.2/topics/class-based-views/generic-display/